### ISSUE TITLE
install.sh should remove naughty httpd conf files

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2201]
_Imported from trac ticket 2201.
Created by **kbower**
Opened in trac: Mar 5, 2013, 4:32:38 PM
Last modified in trac: Mar 5, 2013, 4:57:16 PM_
***
```
manual.conf
perl.conf
php.conf
proxy_ajp.conf
squid.conf
webalizer.conf
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 5, 2013, 4:39:27 PM:_

Alternatively, we could remove files that don't have our "signature" so that we still have flexibility to add files in the future.

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 5, 2013, 4:57:16 PM:_

(Changeset [[3864]](../commit/1e144947b3e9f9930dbed7196dc030261a7d1d4d))
* install.sh should remove potentially interfering httpd conf files (closes #2201)

**Comment by kentbower** [comment]
_**kbower** trac status change on Mar 5, 2013, 4:57:16 PM:_ **closed**
